news/7856.bugfix

```diff
@@ -0,0 +1 @@
+Uninstall should complete successfully, removing the .egg-link, even if the easy-install.pth file is not found.
```

Comment:
Do we need to add an assertion to make sure this is the case?

Since the fix is more general than just dealing with `easy-install.pth` would probably be less specific in the message. Something like *uninstallation no longer fails on trying to remove non-existent files*.

Reply:
That makes sense, Let me fix this quickly

Reply:
Yes I think we can add that, since `easy-install.pth` and `.egg-link` are the two files we remove while `uninstall`. Can we check for `.egg-link` using the approach as below?

https://github.com/pypa/pip/blob/019637cc67039750ba622aa8e842cf1ea8a65a13/tests/functional/test_uninstall.py#L548-L551

Reply:
Okay, I just added that line and the assertion passes successfully, so I can add this assertion to my test

Reply:
Hi @uranusjr,

I have made a commit to address the removal of `egg-link` after installation. Please take a look and let me know if that makes sense :)